bin/kyuubi

```diff
@@ -106,7 +106,20 @@ cmd="${RUNNER} ${KYUUBI_JAVA_OPTS} -cp ${KYUUBI_CLASSPATH} $CLASS"
 
 pid="${KYUUBI_PID_DIR}/kyuubi-$USER-$CLASS.pid"
 
+function get_pid(){
+  if [[ ! -w ${KYUUBI_PID_DIR} ]]; then
+    echo "${USER} does not have 'w' permission to ${KYUUBI_PID_DIR}"
+    exit 1
+  fi
+  
+  run_pid="$(ps -ef | grep kyuubi | grep java | grep -v grep | awk '{print $2}')"
```

Comment:
1. `grep kyuubi` is not sufficient to ensure that this is a Kyuubi Server process. For example, Kyuubi Engine processes also have `kyuubi` in their info.
2. User may have multiple Kyuubi installations on a single machine. We need to ensure that this Kyuubi Server process is launched using current Kyuubi installation.

Reply:
Thanks @zhouyifan279 for your feedback. I will work on these.

Reply:
Hi @zhouyifan279 , I have updated the implementation, now I am making the selection on
1. org.apache.kyuubi.server.KyuubiServer - Ensures its an server process
2. $KYUUBI_CONF_DIR - Kyuubi conf dir path, displayed in the complete process details. This identifier is choosen to differentiate between the instances, because different instances cant run with same conf, as it will cause port already in use error.

Please review the changes.

Thanks